src/index.js

```diff
@@ -1,42 +1,40 @@
-/**************************************************************************
- *  (C) Copyright Mojaloop Foundation 2020 - All rights reserved.         *
- *                                                                        *
- *  This file is made available under the terms of the license agreement  *
- *  specified in the corresponding source code repository.                *
- *                                                                        *
- *  Original Author:                                                      *
- *       Rajiv Mothilal - rajiv.mothilal@modusbox.com                     *
- **************************************************************************/
 'use strict'
 
-const Server = require('./server')
-const PJson = require('../package.json')
-const { Command } = require('commander')
-const Config = require('./lib/config')
-const argv = require('./lib/argv').getArgs()
+const util = require('util')
+const Logger = require('@mojaloop/central-services-logger')
+const Config = require('./config')
+const { createServer } = require('./server')
+const { OracleDatabase } = require('./db')
 
-const Program = new Command()
+const initialize = async () => {
+  // Set log level to info
+  // eslint-disable-next-line
+  Logger.transports.forEach(t => t.level = Config.LOG_LEVEL)
 
-Program
-  .version(PJson.version)
-  .description('CLI to manage Servers')
+  const oracleDb = new OracleDatabase(Config.DATABASE)
+  const server = await createServer(Config.SERVER_PORT, oracleDb)
 
-Program.command('api')
-  .alias('a')
-  .description('Start the Transaction Requests API. Use options to specify server type of none to run both') // command description
+  if (server) {
+    try {
+      // Perform initial health check
+      const unhealthy = await server.app.healthCheck()
+      if (unhealthy) {
+        throw new Error(unhealthy.message)
+      } else {
+        server.log('info', 'Database connected')
+      }
 
-  // function to execute when command is used
-  .action(async () => {
-    const options = {
-      port: Config.PORT
+      server.log('info', `Server running on ${server.info.host}:${server.info.port}`)
+      return server
+    } catch (e) {
+      server.log('error', util.format(e))
+      throw e
     }
-    module.exports = Server.initialize(options.port)
-  })
+  }
+}
 
-if (Array.isArray(argv) && argv.length > 1) {
-  // parse command line vars
-  Program.parse(argv)
-} else {
-  // display default help
-  Program.help()
+try {
+  initialize()
```

Comment:
initialize is an async method, if don't want to wait on a promise to resolve, let leave a comment here why...

Reply:
for now yes to ignore an "Arrow function should not return assignment". this was code I took from the pathfinder repo, so I didn't want to change it up here.